resultsdbupdater/utils.py

```diff
@@ -182,66 +182,67 @@ def ci_metrics_post_to_resultsdb(msg):
 
 def cips_post_to_resultsdb(msg):
     session = retry_session()
+
     # define variables from cips.json
     msg_id = msg['headers']['message-id']
-    ci_type = msg['headers']['ci_type']
+    type = msg['headers']['type']
     component = msg['headers']['component']
-    brew_task_id = msg['headers']['brew_task_id']
+    id = msg['headers']['id']
+    scratch = msg['headers']['scratch']
     msg_body = msg['body']['msg']
-    tests = msg_body['tests']
-
-    arch = msg_body['environment']['arch']
-    brew_tag = msg_body['environment']['brew_tag']
-    build_type = msg_body['environment']['build_type']
-
-    jenkins_job_url = msg_body['infrastructure']['jenkins_job_url']
-    jenkins_build_url = msg_body['infrastructure']['jenkins_build_url']
-
-    cips_report = msg_body['results']['cips_report']
-    cips_status = msg_body['results']['cips_status']
-
-    testcase_url = jenkins_job_url
+    msg_body_ci = msg['body']['msg']['ci']
+    msg_body_artifact = msg['body']['msg']['artifact']
+    url = msg_body['run']['url']
+    outcome = msg_body['status']
+    category = msg_body['category']
+    issuer = msg_body_artifact['issuer']
+    rebuild = msg_body['run']['rebuild']
+    log = msg_body['run']['log']
+    system_os = msg_body['system'][0]['os']
+    system_provider = msg_body['system'][0]['provider']
+    ci_name = msg_body_ci['name']
+    ci_url = msg_body_ci['url']
+    ci_environment = msg_body_ci['environment']
+    ci_team = msg_body_ci['team']
+    ci_irc = msg_body_ci['irc']
+    ci_email = msg_body_ci['email']
 
     # variables to be passed to create_result
     groups = [{
         'uuid': str(uuid.uuid4()),
-        'ref_url': jenkins_build_url
+        'url': url
     }]
-    outcome = cips_status
-    ref_url = jenkins_build_url
+    outcome = outcome
+    url = url
 
     result_data = {
         'item': component,
-        'ci_type': ci_type,
-        'brew_task_id': brew_task_id,
-        'brew_tag': brew_tag,
-        'arch': arch,
+        'type': type,
+        'id': id,
+        'category': category,
         'component': component,
-        'build_type': build_type,
-        'cips_report': cips_report,
-        'cips_status': cips_status,
-        'testcase_url': testcase_url,
+        'scratch': scratch,
+        'status': outcome,
```

Comment:
Seems redundant to have this since the result has an outcome associated with it. It doesn't need to be in the extra metadata.